### PR TITLE
[SPARK-29020][FOLLOWUP][SQL] Update description of array_sort function

### PR DESCRIPTION
**Pull request by kiszk**
### What changes were proposed in this pull request?

This PR is a follow-up of #25728. #25728 introduces additional arguments to determine sort order. Thus, this function does not sort only in ascending order. However, the description was not updated.   
This PR updates the description to follow the latest feature.

### Does this PR introduce any user-facing change?

No

### How was this patch tested?

Existing tests since this PR just updates description text.

### CODE REVIEW

**Comment by kiszk** [comment]
cc @ueshin @srowen

**Review by srowen — APPROVED** [review]

**Comment by srowen** [comment]
Jenkins test this please

File: sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/higherOrderFunctions.scala [hunk 1]
```diff
@@ -291,11 +291,11 @@ case class ArrayTransform(
...
 // scalastyle:off line.size.limit
 @ExpressionDescription(
-  usage = """_FUNC_(expr, func) - Sorts the input array in ascending order. The elements of the
-    input array must be orderable. Null elements will be placed at the end of the returned
-    array. Since 3.0.0 this function also sorts and returns the array based on the given
-    comparator function. The comparator will take two arguments
-    representing two elements of the array.
+  usage = """_FUNC_(expr, func) - Sorts the input array. If func is omitted, sort
+    in ascending order. The elements of the input array must be orderable. Null elements
+    will be placed at the end of the returned. Since 3.0.0 this function also sorts and
```
Comment (MaxGekk) [inline sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/higherOrderFunctions.scala]: Did you skip `array` in _Null elements will be placed at the end of the returned **array**._ intentionally?

Comment (kiszk) [reply sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/higherOrderFunctions.scala]: Good catch. I missed it.

Comment (kiszk) [reply sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/higherOrderFunctions.scala]: Also I realized "and and ...".

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM.

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM. Thank you, @kiszk , @srowen , @MaxGekk .
All tests passed already and the last commit is only touching two words.
Merged to master.

**Pull request closed without merging**